app/views/programming_groups/index.html.slim

```diff
@@ -0,0 +1,23 @@
+h1 = ProgrammingGroup.model_name.human(count: 2)
+
+= render(layout: 'shared/form_filters') do |f|
+  .col-auto
+    = f.label(:exercise_id, t('activerecord.attributes.programming_group.exercise_title'), class: 'visually-hidden form-label')
+    = f.collection_select(:exercise_id, Exercise.with_programming_groups, :id, :title, class: 'form-control', prompt: t('activerecord.attributes.programming_group.exercise_title'))
+
+.table-responsive
+  table.table.mt-4
```

Comment:
Feel free to make the table sortable, by adding:

```
class="#{@programming_groups.present? ? 'sortable' : ''}"
```

Please ensure that sorting in both directions works as expected (will be sorted in the frontend by the client).

Reply:
The sorting is only working in one direction (asc) when clicking on it. In the url it is possible to change it.